### PR TITLE
Use stickers for join and leave actions in non official groups [Fixed commits]

### PR DESCRIPTION
**Pull request by nitanmarcel**
Replace the join message with this sticker:

<details>
<summary>Meow</summary>

image

</details>

Send the following sticker on leave:

<details>
<summary>Meow</summary>

image

</details>

As suggested by @tulir in t.me/telethonofftopic

### CODE REVIEW

**Comment by nitanmarcel** [comment]
The old PR was broken, request changes here if any are needed :)

File: welcome.py [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
-
-    -1001200633650:
-    'Welcome to the off-topic group. Feel free to talk, ask or test anything '
-    'here, politely. Check the description if you need to test more spammy '
-    '"features" of your or other people\'s bots (sed commands too).'
-}
+WELCOME = 'Hi and welcome to the group. Before asking any questions, **please** '\
+          'read [the docs](https://docs.telethon.dev/). Make sure you are '\
+          'using the latest version with `pip3 install -U telethon`, since most '\
+          'problems have already been fixed in newer versions.'
```
Comment (tulir) [inline welcome.py:7]: This should probably stay as a dict so it supports adding more chats if necessary

File: welcome.py [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
                     pass
-
-            last_welcome[event.chat_id] = await event.reply(WELCOME[event.chat_id])
+            if event.chat_id == -1001109500936:
+                last_welcome[event.chat_id] = await event.reply(WELCOME)
+            else:
+                file = await bot.upload_file('hello.webp')
+                last_welcome[event.chat_id] = await event.reply(file=file)
+        if event.user_left or event.user_kicked:
+            if event.chat_id != -1001109500936:
```
Comment (tulir) [inline welcome.py:28]:
```suggestion
            if event.chat_id == -1001200633650:
```

File: welcome.py [hunk 3]
```diff
@@ -1,32 +1,30 @@
...
                 except errors.MessageDeleteForbiddenError:
                     # We believe this happens when trying to delete old messages
                     pass
-
-            last_welcome[event.chat_id] = await event.reply(WELCOME[event.chat_id])
+            if event.chat_id == -1001109500936:
+                last_welcome[event.chat_id] = await event.reply(WELCOME)
+            else:
+                file = await bot.upload_file('hello.webp')
+                last_welcome[event.chat_id] = await event.reply(file=file)
```
Comment (tulir) [inline welcome.py:26]: This is the wrong way around, it should be `if event.chat_id == -1001200633650: image else: welcome`

File: welcome.py (hunk 1 above)
Comment (nitanmarcel) [reply welcome.py:7]: Well the old one didn't had a check to see if the chat Id exists or not in the dict so I assumed there won't be any chats added

File: welcome.py (hunk 3 above)
Comment (nitanmarcel) [reply welcome.py:26]: Hmm -1001109500936 is the on topic chat, why should I send the image there?

**Comment by nitanmarcel** [comment]
So the last commit does that if the chat isn't in the dict it will send the stickers. That way more chats can be added with their own join messages

**Comment by Lonami** [comment]
I think there should be `WELCOME` and `FAREWELL` dictionaries. Main chat would just have a welcome, but offtopic would have the new welcome with the cat sticker, and also be in the farewell with the second cat sticker. Something like:

```python
MAIN_WELCOME = (
    'Hi and welcome to the group. Before asking any questions, **please** '
    'read [the docs](https://docs.telethon.dev/). Make sure you are '
    'using the latest version with `pip3 install -U telethon`, since most '
    'problems have already been fixed in newer versions.'
)

WELCOME = {
    -1001109500936: ((MAIN_WELCOME,), {}),
    -1001200633650: ((), {'file': 'hello.webp'}),
}

FAREWELL = {
    -1001200633650: ((), {'file': 'bye.webp'}),
}
```

Which is definitely overkill but it's the most extensible and clean (rather than sending the file on all chats we're unaware of).

The use of course changes:

```python
welcome = WELCOME.get(event.chat_id)
if welcome:
    args, kwargs = welcome
    last_welcome[event.chat_id] = await event.reply(*args, **kwargs)
```

Same for `FAREWELL`.

**Comment by nitanmarcel** [comment]
@Lonami @tulir done the changes requested :)

**Pull request merged**